torch/csrc/autograd/profiler_utils.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <ATen/ATen.h>
+#include <ATen/record_function.h>
+
+namespace torch { namespace autograd {
+namespace profiler {
+
+std::unordered_map<std::string, c10::IValue> TORCH_API saveExtraArgs(const at::RecordFunction& fn);
+
+uint64_t TORCH_API computeFlops(const at::StringView &op_name,
```

Comment:
could you also remove the use of StringView here and in profiler_utils.cpp?

Reply:
Do you mean we replace at::StringView with std::string? If so, we also need to pass std::string in profiler_legacy.cpp.

Reply:
yes, let's not use StringView in public api